xbmc/pictures/metadata/ImageMetadataParser.cpp

```diff
@@ -75,27 +83,28 @@ std::unique_ptr<ImageMetadata> CImageMetadataParser::ExtractMetadata(const std::
   CImageMetadataParser parser;
 
   // extract metadata
-  parser.ExtractCommonMetadata(image);
+  parser.ExtractCommonMetadata(*image);
 
   parser.ExtractExif(image->exifData());
   parser.ExtractIPTC(image->iptcData());
 
   return std::move(parser.m_imageMetadata);
 }
 
-void CImageMetadataParser::ExtractCommonMetadata(std::unique_ptr<Exiv2::Image>& image)
+void CImageMetadataParser::ExtractCommonMetadata(Exiv2::Image& image)
 {
   //! TODO: all these elements are generic should be moved out of the exif struct
-  m_imageMetadata->height = image->pixelHeight();
```

Comment:
TODO above is a leftover, I can clean it later. Members were already moved out of the exif struct